src/Lecture1.hs

```diff
@@ -104,7 +112,17 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+strSum :: [Char] -> Integer
+strSum str = 
+    let list = words str
+        summ = 0
+    in go list summ
+        where
+            go :: [[Char]] -> Integer -> Integer
+            go list summ = 
+                if null list
+                then summ
+                else go (drop 1 list) (summ + read (head list) :: Integer)
```

Comment:
This is the correct implementation 👍🏻 

I see that you've implemented this using manual recursion. It's cool that you use it fluently 😎 
Here you can simplify the implementation by using standard functions `map` and `sum`